src/PluginConfig.cs

```diff
@@ -454,8 +454,8 @@ static PluginConfig InitLegacyDefault(IPluginHost host)
 			if (!Version.TryParse(verString, out configVersion))
 			{
 				configVersion = new Version(Ver0);
-				update.ConfigVersion = Ver0;
 			}
+			update.ConfigVersion = configVersion;
```

Comment:
This doesn't compile, try:
`
			update.ConfigVersion = configVersion.ToString(2);
`